src/network.js

```diff
@@ -480,15 +480,23 @@ export class BlockstackNetwork {
       reverseHash.reverse()
       excludeSet.push({
         tx_hash: reverseHash.toString('hex'),
-        tx_output_n: utxoUsed.index 
+        tx_output_n: utxoUsed.index
       })
     })
 
     this.excludeUtxoSet = excludeSet
 
     const txHash = tx.getHash().reverse().toString('hex')
     tx.outs.forEach((utxoCreated, txOutputN) => {
-      if (bitcoinjs.script.classifyOutput(utxoCreated.script) === 'nulldata') {
```

Comment:
classifyOutput is not exposed anymore so we need another way to figure out if its nulldata